connectors/sources/outlook.py

```diff
@@ -735,6 +823,13 @@ def get_default_configuration(cls):
                 "sensitive": True,
                 "type": "str",
             },
+            "client_emails": {
+                "depends_on": [{"field": "data_source", "value": OUTLOOK_CLOUD}],
+                "label": "Client Email Addresses (comma-separated)",
+                "order": 5,
+                "required": False,
+                "type": "str",
```

Comment:
I'd expect this to be a list. The framework will then do the "split on comma" work for you. See:
https://github.com/elastic/connectors/blob/6e74c3a8023f3fd80de240e5ffb446321610225f/connectors/source.py#L142-L144